layouts/shortcodes/admin-users.html

```diff
@@ -32,16 +32,14 @@
 * **Account Created**: The time and date when the user account was created.` }}
 {{ $member_navigation = "Select your company in the left navigation drop-down menu, and then select **Users**." }}
 {{ $remove_button = "**Remove user**" }}
-{{ $update_role = "Select their organization, select the role you want to assign, and then select **Save**." }}
+Select their organization, select the role you want to assign, and then select **Save**.{{ $update_role = "" }}
```

Comment:
What's going on with this change?

Reply:
I think I meant to delete the update role shortcode since I pulled the info into the markdown file, and I had copy/paste mis-type *face palm*